react/features/invite/components/InviteButton.web.js

```diff
@@ -0,0 +1,231 @@
+import Button from '@atlaskit/button';
+import DropdownMenu from '@atlaskit/dropdown-menu';
+
+import { translate } from '../../base/i18n';
+import { getLocalParticipant, PARTICIPANT_ROLE } from '../../base/participants';
+
+import { openDialog } from '../../base/dialog';
+import { AddPeopleDialog, InviteDialog } from '.';
+import { DialOutDialog } from '../../dial-out';
+import { isInviteOptionEnabled, getInviteOptionPosition } from '../functions';
+
+declare var interfaceConfig: Object;
+
+const SHARE_LINK_OPTION = 'invite';
+const DIAL_OUT_OPTION = 'dialout';
+const ADD_TO_CALL_OPTION = 'addtocall';
+
+/**
+ * The button that provides different invite options.
+ */
+class InviteButton extends Component {
+    /**
+     * {@code InviteButton}'s property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Indicates if the "Add to call" feature is available.
+         */
+        _isAddToCallAvailable: PropTypes.bool,
+
+        /**
+         * Indicates if the "Dial out" feature is available.
+         */
+        _isDialOutAvailable: PropTypes.bool,
+
+        /**
+         * The function opening the dialog.
+         */
+        openDialog: PropTypes.func,
+
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: PropTypes.func
+    };
+
+    /**
+     * Initializes a new {@code InviteButton} instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     */
+    constructor(props) {
+        super(props);
+
+        this._onInviteClick = this._onInviteClick.bind(this);
+        this._onInviteOptionSelected = this._onInviteOptionSelected.bind(this);
+        this._updateInviteItems = this._updateInviteItems.bind(this);
+
+        this._updateInviteItems(this.props);
+    }
+
+    /**
+     * Implements React's {@link Component#componentWillReceiveProps()}.
+     *
+     * @inheritdoc
+     * @param {Object} nextProps - The read-only props which this Component will
+     * receive.
+     * @returns {void}
+     */
+    componentWillReceiveProps(nextProps) {
+        if (this.props._isDialOutAvailable !== nextProps._isDialOutAvailable
+            || this.props._isAddToCallAvailable
+                !== nextProps._isAddToCallAvailable) {
+            this._updateInviteItems(nextProps);
+        }
+    }
+
+    /**
+     * Renders the content of this component.
+     *
+     * @returns {ReactElement}
+     */
+    render() {
+        const { t } = this.props;
+
+        const { VERTICAL_FILMSTRIP } = interfaceConfig;
+
+        return (
+            <div className = 'filmstrip__invite'>
+                <div className = 'invite-button-group'>
+                    <Button
+                        onClick = { this._onInviteClick }
+                        shouldFitContainer = { true }>
+                        { t('invite.invitePeople') }
+                    </Button>
```

Comment:
You might want to add truncation of the text if possible with Atlaskit, in case some translation makes the button stretch horizontally. Maybe use max width on the containing element as a backup option for preventing horizontal stretch?